sdk/storage/azure-storage-files-shares/inc/azure/storage/files/shares/protocol/share_rest_client.hpp

```diff
@@ -7264,13 +7265,13 @@ namespace Azure { namespace Storage { namespace Files { namespace Shares {
           }
           else if (node.Type == XmlNodeType::Text)
           {
-            if (path.size() == 1 && path[0] == XmlTagName::c_Start)
+            if (path.size() == 1 && path[0] == XmlTagName::c_End)
             {
-              result.Start = std::stoll(node.Value);
+              result.End = std::stoll(node.Value);
             }
-            else if (path.size() == 1 && path[0] == XmlTagName::c_End)
+            else if (path.size() == 1 && path[0] == XmlTagName::c_Start)
             {
-              result.End = std::stoll(node.Value);
+              result.Start = std::stoll(node.Value);
```

Comment:
Same here/elsewhere in this file. Don't seem relevant to Url and Url encoding.

Reply:
Same as the other reply.

Reply:
@katmsft You should've created a separate PR.